instance/static/html/instance/index.html

```diff
@@ -24,6 +24,13 @@
       <li ng-repeat="instance in instanceList"
           ng-class="{active: instance == selected.instance}">
         <a ng-click="select('instance', instance)">{{ instance.name }}</a>
+        <div class="status-icon" ng-switch="instance.status">
+          <i class="fa fa-ellipsis-h" tooltip-placement="top" tooltip="Provisioning in progress" ng-switch-default></i>
+        </div>
```

Comment:
I think it is not clear to me whether to display the "Provisioning in progress" text/icon. It seems like the way you saying we will always display it, but what if there's no provisioning happening? For example, when you just created an instance and didn't start the provisioning process?